crates/store/re_protos/proto/rerun/v0/log_msg.proto

```diff
@@ -0,0 +1,178 @@
+syntax = "proto3";
+
+package rerun.log_msg.v0;
+
+import "rerun/v0/common.proto";
+
+// Corresponds to `LogMsg::SetStoreInfo`. Used to identify a recording.
+message SetStoreInfo {
```

Comment:
I was hoping that ``SetStoreInfo`` would go away, mostly for selfish reasons of exposing ``GetStoreInfo`` on the SN side that would return ``StoreInfo``. Reading the comment on the ``row_id`` field of the current ``SetStoreInfo`` struct doesn't help me. 

@teh-cmc  some help here, do we need this still?

Reply:
Removing `StoreInfo` is a bit complicated, but it will eventually be done. Currently we may have multiple recordings per `rrd` file, and changing that would have far-reaching consequences that I wasn't comfortable with for this PR.

Reply:
Yes sorry, I meant `SetStoreInfo`. Its current purpose is to support multiple recordings per rrd. That's pretty much it.

The way this should look in the end is outlined in Notion: https://www.notion.so/rerunio/Protobufication-143b24554b19803e83acd5cc34bd5219?pvs=4#143b24554b19801cad70ce9327a2f651 - but there are no concrete plans on how to get there. And I'll definitely need help figuring that out :smile: